kubernetes/update-stack.sh

```diff
@@ -28,6 +28,22 @@ update_runplugin() {
     ln -f "${WAGGLE_BIN_DIR}/runplugin-${arch}" "${WAGGLE_BIN_DIR}/runplugin"
 }
 
+update_node_secrets() {
+    (
+    if [ -e /root/.ssh/node-private-git-repo-key ] ; then
+        export GIT_SSH_COMMAND='ssh -i /root/.ssh/node-private-git-repo-key -o StrictHostKeyChecking=no -o IdentitiesOnly=yes'
+        if [ -e /opt/node-config-private ] ; then
```

Comment:
In the past what I have done just to ensure it works every time is save these things in a tmpfs (/tmp or /run) and then delete when I am done.  that way any failures in clone or pull or system reboots will start you from scratch.

just a thought

Reply:
@jswantek That makes sense. I was worried about the traffic our node will generate because this happens quite often -- once every 10 minutes per node. If this does not harm the system, I would like to keep it as is to save some traffic potentially.

Reply:
that is fine.  just sharing some knowledge. :)